MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/App_Start/WebApiConfig.cs

```diff
@@ -1,18 +1,25 @@
 ﻿using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
 
 namespace MyPerfectOnboarding.Api
 {
     public static class WebApiConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            config.MapHttpAttributeRoutes();
+            var constraintResolver = new DefaultInlineConstraintResolver()
+            {
+                ConstraintMap =
+                {
+                    ["apiVersion"] = typeof( ApiVersionRouteConstraint )
+                }
+            };
+            config.MapHttpAttributeRoutes(constraintResolver);
+            config.AddApiVersioning();
 
-            config.Routes.MapHttpRoute(
-                name: "DefaultApi",
-                routeTemplate: "api/{controller}/{id}",
-                defaults: new { id = RouteParameter.Optional }
-            );
+            var json = GlobalConfiguration.Configuration.Formatters.JsonFormatter;
```

Comment:
Move setting the json to new file, something like JsonFormatterConfig would be appropriate. It should work same as WebApiConfig - it will have Register method with HttpConfiguration config and will be called from same place as WebApiConfig is called.